determined-ai/determined
# chore: Use Hew SplitPane [WEB-1682]

## Description

Removes `SplitPane` from Determined and replaces it with the Hew `SplitPane`




## Test Plan
1. Navigate to the Experiment List page. One way to do so is by clicking on "Uncategorized" on the left sidebar.
2. Click the "Compare" button on the right side of the screen above the table and choose at least one experiment. Ensure that the "Compare" tab opens and closes.
3. With the "Compare" tab open, use the divider to change the width of each pane. 
4. Refresh the page and ensure that the Pane widths are the same as they were when they were changed in (3).
5. Switch to a mobile view. This can be done by going to Devtools/Inspect Element (on Mac in Safari by pressing "CTRL + Right Click" on the webpage and then clicking inspect) then on the top left of the DevTools window clicking the laptop and phone icon. 
![788E23A4-26F8-49D9-BD5F-33A3907E7ABD_4_5005_c](https://github.com/determined-ai/determined/assets/103522725/769f3b5b-1894-471a-b692-64aec77641b7)
then set the view to a small mobile dimension, such as 375 x 667. 
6. Ensure that when the Compare option is not enabled only the experiment list is visible.
7. Ensure that when the Compare option is enabled only the compare view is visible.






## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
WEB-1682
